wiki/Glossary/en.md

```diff
@@ -309,6 +309,10 @@ A term used when a player completes a beatmap while obtaining the maximum combo
 
 Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in the osu!standard, osu!taiko, and osu!catch modes.
 
+### Featured Artist
+
+A featured artist is someone who has officialy licensed music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping.
```

Comment:
```suggestion
*See also: [Featured Artists](https://osu.ppy.sh/beatmaps/artists)*

A featured artist is an artist who has officially licensed their music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping. All tracks from featured artists include a pre-timed `.osz` file to help you focus on beatmapping.
```

Reply:
any reason for website link instead of wiki ?

Reply:
that should probably go into the description

Reply:
@cl8n fwiw now, I thought there wasn't a wiki page for it